explore/src/main/scala/explore/attachments/ObsAttachmentsTableBody.scala

```diff
@@ -323,54 +327,17 @@ object ObsAttachmentsTable extends ObsAttachmentUtils:
             ).sortableBy(_.get.checked)
           )
       // Rows
-      .useMemoBy((props, _, _, _, _, _) => props.obsAttachments.reuseByValue): (_, _, _, _, _, _) =>
+      .useMemoBy((props, _, _, _, _) => props.obsAttachments.reuseByValue): (_, _, _, _, _) =>
         _.value.toListOfViews.map(_._2)
-      .useReactTableBy: (props, _, client, _, urlMap, cols, rows) =>
+      .useReactTableBy: (props, _, client, urlMap, cols, rows) =>
         TableOptions(
           cols,
           rows,
           getRowId = (row, _, _) => RowId(row.get.id.toString),
           meta = TableMeta(client, props.obsAttachmentAssignments, urlMap.get, props.readOnly)
         )
-      .useStateView(Enumerated[ObsAttachmentType].all.head)
-      .render: (props, ctx, client, action, _, _, _, table, newAttType) =>
-        import ctx.given
-
+      .render: (props, _, _, _, _, _, table) =>
         React.Fragment(
-          props.renderInTitle(
```

Comment:
This is an example where a new component for the title was extracted out from the body and it is now an independent component for the title